### PR TITLE
Allow to translate placeholder in create account section

### PR DESCRIPTION
**Pull request by ghost**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop 
| Description?      | Allow to translate date format placeholder
| Type?             | improvement
| Category?         | FO
| BC breaks?        | no
| Deprecations?     | yes
| Fixed ticket?     | 
| Related PRs       | If theme, autoupgrade or other module change is needed, provide a link to related PRs here.
| How to test?      | Test if you can translate date format placeholder in create customer account
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

File: classes/Tools.php [hunk 1]
```diff
@@ -941,12 +941,22 @@ public static function displayDate($date, $full = false)
...
+     *
      * Get localized date format.
      *
      * @return string Date format
      */
     public static function getDateFormat()
     {
+        @trigger_error(
+            sprintf(
+                '%s is deprecated since version 8.1.0',
```
Comment (Progi1984) [inline classes/Tools.php]:
```suggestion
                '%s is deprecated since version 8.1.0' and will be removed in the next major version.,
```

**Review by Progi1984 — DISMISSED** [review]

File: classes/Tools.php [hunk 2]
```diff
@@ -941,12 +941,22 @@ public static function displayDate($date, $full = false)
...
+     *
      * Get localized date format.
      *
      * @return string Date format
      */
     public static function getDateFormat()
     {
+        @trigger_error(
+            sprintf(
+                '%s is deprecated since version 8.1.0' and will be removed in the next major version.,
```
Comment (Progi1984) [inline classes/Tools.php]:
```suggestion
                '%s is deprecated since version 8.1.0 and will be removed in the next major version.',
```

**Review by Progi1984 — DISMISSED** [review]

File: classes/form/CustomerFormatter.php [hunk 3]
```diff
@@ -209,7 +209,7 @@ public function getFormat()
                         'Shop.Forms.Labels'
                     )
                 )
-                ->addAvailableValue('placeholder', Tools::getDateFormat())
+                ->addAvailableValue('placeholder', $this->translator->trans('DD/MM/JJJJ',[] , 'Shop.Theme.Customeraccount'))
```
Comment (PrestaEdit) [inline classes/form/CustomerFormatter.php]:
```suggestion
                ->addAvailableValue('placeholder', $this->translator->trans('DD/MM/YYYY',[] , 'Shop.Theme.Customeraccount'))
```

File: classes/form/CustomerFormatter.php [hunk 4]
```diff
@@ -209,7 +209,7 @@ public function getFormat()
                         'Shop.Forms.Labels'
                     )
                 )
-                ->addAvailableValue('placeholder', Tools::getDateFormat())
+                ->addAvailableValue('placeholder', $this->translator->trans('DD/MM/JJJJ', [], 'Shop.Theme.Customeraccount'))
```
Comment (jf-viguier) [inline classes/form/CustomerFormatter.php]:
```suggestion
                ->addAvailableValue('placeholder', $this->translator->trans('DD/MM/YYYY', [], 'Shop.Theme.Customeraccount'))
```

Comment (jf-viguier) [reply classes/form/CustomerFormatter.php]: is JJJJ a new kind of troll ?

Comment (kpodemski) [inline classes/form/CustomerFormatter.php]: @okom3pom I think it should be `DD/MM/YYYY`?

Comment (ghost) [reply classes/form/CustomerFormatter.php]: oups

Comment (ghost) [reply classes/form/CustomerFormatter.php]: Fixed @kpodemski

Comment (PrestaEdit) [reply classes/form/CustomerFormatter.php]: It was the goal of my suggestion, ;p

File: classes/form/CustomerFormatter.php (hunk 3 above)
Comment (PrestaEdit) [reply classes/form/CustomerFormatter.php]: @okom3pom It's seems I can't do it, maybe you can "Resolve" this with the related button :)

**Review by kpodemski — DISMISSED** [review]

**Pull request closed without merging**